plugin.rb

```diff
@@ -41,7 +41,8 @@
   end
 
   add_to_class(:guardian, :can_create_jira_issue?) do
-    SiteSetting.discourse_jira_enabled && is_staff?
+    SiteSetting.discourse_jira_enabled &&
+      user&.in_any_groups?(SiteSetting.discourse_jira_allowed_groups_map)
```

Comment:
What would happen if `discourse_jira_allowed_groups_map` is an empty array?

Reply:
I can add a test, but isn't that impossible considering: https://github.com/discourse/discourse-jira/blob/e19854f4c43177e1fb23d3c3f0605b5b343b9308/config/settings.yml @ line 28 with the `mandatory_values` setting?

Reply:
Ah I see. Then it should be good. 👍